proto/info/info.proto

```diff
@@ -1143,6 +1157,8 @@ message ZInfoMsg {
     ZInfoHardware hwinfo = 18;
     ZInfoLocation locinfo = 19;
     ZInfoPatchEnvelope patchInfo = 20;
+    // 21 reserved
```

Comment:
What was 21 used for?

Reply:
Don't know! the `reserved` statement is much earlier in the message definition.  The comment is on the edge of visual noise, added it to highlight a statement that is easy to miss ;)

blame:
```
27069fa6 proto/info/info.proto     (Pavel Abramov           2023-10-30 17:09:38 +0100 1142)   reserved 21;
```
commit record:
```
commit 27069fa6864fd403e6bf2b48c38dd34b17e48768
Author: Pavel Abramov <uncle.decart@gmail.com>
Date:   Mon Oct 30 17:09:38 2023 +0100

    PatchEnvelopes: Flatten `ZInfoPatchEnvelopeUsage` and remove Opaque Status
    
    There is no need for additional structure since there is
    1 to 1 releation. This commit also removes opaque status since it is
    duplication of AppInstMetaData
    
    Signed-off-by: Pavel Abramov <uncle.decart@gmail.com>
```

Reply:
Yes, Pavel removed `ZInfoOpaqueAppInstanceStatus opaqueAppInstStatus = 21` in the commit referenced by Bob.

Reply:
@famleebob I assume that if you accidentally tried to reuse 21, given the reserved statement the protoc compiler should have given an error. Did it not?

Reply:
Yes, it did.  The `reserved` statement gets lost in the text (at least for me).  I added the comment in its where someone might try to add `21`.  I can remove it if you wish.